src/dom-utils.ts

```diff
@@ -37,3 +39,35 @@ export function updateZIndex() {
 function isHTMLElement(node: Node): node is HTMLElement {
   return node.nodeType === Node.ELEMENT_NODE;
 }
+
+const applyIntpusMutations = (muts: PreEngagementFormMutations) => {
```

Comment:
```suggestion
const applyInputMutations = (muts: PreEngagementFormMutations) => {
```
?

Reply:
`Intpus`.. Well, that's the proof I didn't c&p that one :laughing: